rust/src/countme.rs

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (C) 2020 Red Hat, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+use anyhow::{bail, Result};
+use curl::easy::Easy;
+use os_release::OsRelease;
+use std::path;
+
+mod cookie;
+mod repo;
+
+// Default variant name used in User Agent
+const DEFAULT_VARIANT_ID: &str = "unknown";
+
+// Send a GET request to 'url' with 'ua' as User Agent and discard the response
```

Comment:
Minor: did we ever find out BTW if `HEAD` is enough? That way we can make this even more lightweight. :)

Reply:
Asked about that to Fedora admins